x-pack/test/security_solution_endpoint_api_int/apis/artifacts/index.ts

```diff
@@ -22,7 +19,6 @@ export default function (providerContext: FtrProviderContext) {
   let agentAccessAPIKey: string;
 
   describe('artifact download', () => {
-    setupIngest(providerContext);
```

Comment:
This isn't needed because `setup` is called before all the tests in this directory initially.